internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,7 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	prettyJSON, _ := json.MarshalIndent(sb, "", "\t")
+	csv.GetAnnotations()["alm-examples"] = "|" + "\n" + string(prettyJSON)
```

Comment:
I'm guessing this won't work. I think we need to set the annotation to the actual string we want since the YAML marshaler is the thing that knows how to convert and format our structs as YAML. It might be enough to just set this to `string(prettyJSON)`, but we would need to test it.

This is what I meant when I said the YAML Marshaller may not support this. But let's hope it does :crossed_fingers: 

```suggestion
	csv.GetAnnotations()["alm-examples"] = string(prettyJSON)
```

Reply:
Hi @joelanford really tks for the help. It is not working yet. :-) 
No worry I could test it locally now. :-) So, I will check it and I will let you know when I finish.